packages/workbox-streams/src/strategy.ts

```diff
@@ -7,22 +7,26 @@
 */
 
 import {logger} from 'workbox-core/_private/logger.js';
-
 import {createHeaders} from './utils/createHeaders.js';
 import {concatenateToResponse} from './concatenateToResponse.js';
 import {isSupported} from './isSupported.js';
-
+import {handlerCallback, HandlerCallbackOptions} from 'workbox-routing/_types.js';
+import {StreamSource} from './_types.js';
 import './_version.js';
 
+
+type streamsHandlerCallback =
+    ({url, request, event, params}: HandlerCallbackOptions) =>
+        Promise<StreamSource> | StreamSource;
+
 /**
  * A shortcut to create a strategy that could be dropped-in to Workbox's router.
  *
  * On browsers that do not support constructing new `ReadableStream`s, this
  * strategy will automatically wait for all the `sourceFunctions` to complete,
  * and create a final response that concatenates their values together.
  *
- * @param {
- *   Array<function(workbox.routing.Route~handlerCallback)>} sourceFunctions
```

Comment:
@jeffposnick I believe this is incorrect currently, so I've changed it in this PR. But just wanted you to double check to confirm.

Reply:
Here's an example of `workbox.streams.strategy` being used in the wild:

https://github.com/GoogleChromeLabs/so-pwa/blob/9385b059bb5265735525e4133eb3c4424ecba725/src/service-worker.mjs#L75-L92

I think based on that, yeah, `Array<handlerCallback>` is the appropriate type.

Reply:
Yeah, and here's another example: https://github.com/philipwalton/blog/blob/23a69017694e8b63332f0cd42cefd92a7f8b287c/assets/sw/routes/page.js#L25-L29.